src/jinterface.jl

```diff
@@ -0,0 +1,69 @@
+"""
+Creates a type which incorporates the mesh attributes alongside the necessary values
+to solve finite element problems. It works by taking in a FEniCS mesh, working out
+the necessary values, and then creating an ordering of the internal/boundary nodes
+and applying the DirichletBC (via a function) to the respective nodes.
+"""
+type feMesh
+    n_nodes::Int64
```

Comment:
Is 64 for C-operability? Then you might want to use `CInt`. If on 32-bit systems it should use 32-bit integers (which is possibly the case?), then use `Int`. If FEniCS doesn't work on 32-bit, then none of this matters but for style I would use `Int`.

Reply:
Will change to Int to allow operability on both.